docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1258,6 +1258,44 @@ Renames an existing field. If the field doesn't exist or the new name is already
 }
 --------------------------------------------------
 
+[[script-processor]]
+=== Script Processor
+
+Script Processor allows inline, stored, and file scripts to be executed within Ingest Pipelines.
```

Comment:
To be consistent with other topics, start with a verb and do not capitalize ingest pipelines:

"Allows inline, stored, and file scripts to be executed within ingest pipelines."